pkg/api/common.go

```diff
@@ -101,6 +102,7 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	cfg.Auth = &Auth{}
 
 	fs.BoolVar(&cfg.ReadOnly, "read-only", false, "Read-only mode for the connector. Operations related to writing or updating the database are disallowed. It is used when pointing the connector to a TimescaleDB read replica.")
+	fs.BoolVar(&cfg.HighAvailability, "high-availability", false, "Enable external_labels based HA.")
```

Comment:
let's leave a note about the cli rename in the draft release nnotes

Reply:
I think its too late for the experimental part now, it should have been a part of the previous release. Its going to go out of experimental by next release and adding it now doesn't make any sense.

Reply:
yeah no experimental message